addons/rating/models/rating.py

```diff
@@ -55,9 +70,20 @@ def _compute_rating_image(self):
     @api.multi
     @api.depends('rating')
     def _compute_rating_text(self):
-        text = {10: _('Satisfied'), 5: _('Not satisfied'), 1: _('Highly dissatisfied')}
+        text = {10: _('satisfied'), 5: _('not_satisfied'), 1: _('highly_dissatisfied')}
```

Comment:
I also think we should use the same repartition than in `rating_get_grades` (see https://github.com/odoo/odoo/blob/master/addons/rating/models/rating.py#L237)